lib/LTP/utils.pm

```diff
@@ -94,6 +94,7 @@ sub log_versions {
     my $kernel_pkg_log = '/tmp/kernel-pkg.txt';
     my $ver_linux_log  = '/tmp/ver_linux_before.txt';
     my $kernel_config = script_output('for f in "/boot/config-$(uname -r)" "/usr/lib/modules/$(uname -r)/config" /proc/config.gz; do if [ -f "$f" ]; then echo "$f"; break; fi; done');
+    my $is_install_ltp = (((caller 1)[3] // '') =~ /^install_ltp/);
```

Comment:
Please don't do this. Add a parameter which controls the softfail instead.

Reply:
ok